resources/de-amp.js

```diff
@@ -3,6 +3,18 @@
     const attr = 'jsaction'
     const selector = `a[data-amp-title][${attr}]`
     let timer
+    const idleCallback = (_ => {
+      if (self.requestIdleCallback) {
+        return (callback, timeout) => {
+          return self.requestIdleCallback(callback, { timeout })
+        }
+      }
+
+      // Fallback to setTimeout when requestIdleCallback doesn't exist.
+      return (callback, timeout) => {
+        return self.setTimeout(callback, timeout)
```

Comment:
I dont think we want to use the timeout here, since it'll force the worst case time, when what we want is "the soonest available event loop slot".  A very low value would be better here (0, 1, something like that)

Reply:
Done